helm/examples/efs/hpcc-efs/templates/hpcc-efs-server.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "efs-provisioner-%s" (include "hpcc-efs.fullname" $) }}
+data:
+  file.system.id: {{ .Values.efs.id }}
+  aws.region: {{ .Values.efs.region }}
+  provisioner.name: aws.io/aws-efs
+  dns.name: ""
+---
+kind: Deployment
+#apiVersion: extensions/v1beta1
+apiVersion: apps/v1
+metadata:
+  name: {{ printf "efs-provisioner-%s" (include "hpcc-efs.fullname" $) }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ printf "efs-provisioner-%s" (include "hpcc-efs.fullname" $) }}
+  template:
+    metadata:
+      labels:
+        app: {{ printf "efs-provisioner-%s" (include "hpcc-efs.fullname" $) }}
+    spec:
+      containers:
+        - name: {{ printf "efs-provisioner-%s" (include "hpcc-efs.fullname" $) }}
+          #image: quay.io/external_storage/efs-provisioner:latest
+          image: quay.io/external_storage/efs-provisioner:v2.2.0-k8s1.12
```

Comment:
What are the characteristics of this image?
The README should probably explain, reference it.
Might be good to have an option for the image version in values.yaml

Reply:
It is a third party image which is used in AWS EFS example.
I make image as a value in values.yaml.